packages/grafana-runtime/src/utils/queryResponse.ts

```diff
@@ -30,17 +39,24 @@ interface DataResponse {
  *
  * @public
  */
-export function toDataQueryResponse(res: any, queries?: DataQuery[]): DataQueryResponse {
+export function toDataQueryResponse(
+  res:
```

Comment:
The typing is super ugly but seems like this functions just tries to process a lot of different things.